mono/mini/mini-exceptions.c

```diff
@@ -1500,13 +1500,7 @@ ves_icall_get_frame_info (gint32 skip, MonoBoolean need_file_info,
 			case FRAME_TYPE_MANAGED_TO_NATIVE:
 			case FRAME_TYPE_DEBUGGER_INVOKE:
 			case FRAME_TYPE_TRAMPOLINE:
-				continue;
 			case FRAME_TYPE_INTERP_TO_MANAGED:
-				if (!ji) {
-					/* gsharedvt_out_sig_wrapper from interp2jit transition */
-					skip--;
-					break;
```

Comment:
this was introduced by https://github.com/mono/mono/commit/30ad21953397d96d4017286f905794ad4adabbdb not exactly sure if we can remove it, maybe your changes around wrappers make it work for those regression tests though

Reply:
I would need to double check, but the skip should be done by the new interp frame for the added wrapper